Assets/Scripts/ObstacleSpike.cs

```diff
@@ -0,0 +1,19 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class ObstacleSpike : MonoBehaviour
+{
+	private void Start(){
+		
+	}
+
+	public void OnCollisionEnter2D(Collision2D collision){
+
+		if(collision.gameObject.tag == "Player"){
```

Comment:
Would look good with some spaces before the parenthesis and bracket. Same with brackets on functions